reqs/build.pip

```diff
@@ -1,4 +1,4 @@
-numpy<1.20; platform_machine != "arm64"
+numpy
```

Comment:
Should I add versioning requirements here, e.g. `numpy>=1.21.3,<2`? The other dependencies (aside from `protobuf`) don’t have versioning requirements.